single_tech_samples/databricks/databricks_ci_cd/devops/template/test-spark-python-job.yml

```diff
@@ -66,7 +66,7 @@ jobs:
 
         - script: |
             echo "dbx launch job $DATABRICKS_JON_NAME ..."
-            dbx launch --job=$DATABRICKS_JON_NAME --as-run-submit --trace
+            dbx launch $DATABRICKS_JON_NAME --as-run-submit --trace
```

Comment:
Hi @chenshu,
The PR looks good! Thanks for your help to make this single tech sample up-to-date!
Just a minor issue, there's an legacy typo in variable name _$DATABRICKS_**JON**_NAME_, would you please help to correct them in all relevant occurrences, thanks a lot! :- )
Best,
Thurston

Reply:
Done! Thank you!